src/solvers/dg/dg_2d.jl

```diff
@@ -363,61 +369,216 @@ function calc_volume_integral!(du::AbstractArray{<:Any,4}, u, nonconservative_te
     split_form_kernel!(du, u, nonconservative_terms, equations, volume_flux_dg, dg, cache, element, 1 - alpha_element)
 
     # Calculate FV volume integral contribution
-    fv_kernel!(du, u, equations, volume_flux_fv, dg, cache, element, alpha_element)
+    fv_kernel!(du, u, nonconservative_terms, equations, volume_flux_fv, dg, cache, element, alpha_element)
   end
 
   return nothing
 end
 
-@inline function fv_kernel!(du::AbstractArray{<:Any,4}, u::AbstractArray{<:Any,4},
+@inline function fv_kernel!(du::AbstractArray{<:Any,4}, u::AbstractArray{<:Any,4}, nonconservative_terms,
                             equations, volume_flux_fv, dg::DGSEM, cache, element, alpha=true)
-  @unpack fstar1_threaded, fstar2_threaded = cache
+#  @unpack fstar1_threaded, fstar2_threaded = cache
+  @unpack fstar1_L_threaded, fstar1_R_threaded, fstar2_L_threaded, fstar2_R_threaded = cache
   @unpack inverse_weights = dg.basis
 
   # Calculate FV two-point fluxes
-  fstar1 = fstar1_threaded[Threads.threadid()]
-  fstar2 = fstar2_threaded[Threads.threadid()]
-  calcflux_fv!(fstar1, fstar2, u, equations, volume_flux_fv, dg, element)
+  # fstar1 = fstar1_threaded[Threads.threadid()]
+  # fstar2 = fstar2_threaded[Threads.threadid()]
+  # calcflux_fv!(fstar1, fstar2, u, equations, volume_flux_fv, dg, element)
+
+  fstar1_L = fstar1_L_threaded[Threads.threadid()]
+  fstar2_L = fstar2_L_threaded[Threads.threadid()]
+  fstar1_R = fstar1_R_threaded[Threads.threadid()]
+  fstar2_R = fstar2_R_threaded[Threads.threadid()]
+  calcflux_fv!(fstar1_L, fstar1_R, fstar2_L, fstar2_R, u, equations, volume_flux_fv, dg, element,
+               nonconservative_terms)
```

Comment:
```suggestion
  calcflux_fv!(fstar1_L, fstar1_R, fstar2_L, fstar2_R, u, 
               nonconservative_terms, equations, volume_flux_fv, dg, element)
```
I would prefer to keep the order of arguments `nonconservative_terms, equations` as (hopefully) everywhere else. I tried to use the rule to keep arguments specifying another argument directly in front of it, e.g. `nonconservative_terms, equations`. If you make this change, you need to adapt the other parts below, of course.